### DISCUSSION TITLE
JavaScript code templates

### DISCUSSION THREAD

**Discussion by dinkoivanov-gepard** [mickaelistria/eclipse-bluesky#1456] [Q&A] [answered]
Are there any JavaScript code templates in Eclipse Wildwebdeveloper?
Like the Java Code templates in JDT.
For example, a code template for a switch-case statement of a for cycle.

Thanks,
Dinko

**Comment by mickaelistria** [comment, marked as answer]
No, there are not. Such code templates would be best directly in the typescript-language-server.

**Reply by dinkoivanov-gepard** [reply]
Thanks Mickael.
